apps/src/aichat/views/presentation/PresentationView.tsx

```diff
@@ -10,12 +10,12 @@ import moduleStyles from './presentation-view.module.scss';
 import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 
 const PresentationView: React.FunctionComponent = () => {
-  const currentAiCustomizations = useAppSelector(
-    state => state.aichat.currentAiCustomizations
+  const previouslySavedAiCustomizations = useAppSelector(
+    state => state.aichat.previouslySavedAiCustomizations
   );
   const {systemPrompt, temperature, retrievalContexts} =
-    currentAiCustomizations;
-  const modelCardInfo = currentAiCustomizations.modelCardInfo;
+    previouslySavedAiCustomizations;
+  const modelCardInfo = previouslySavedAiCustomizations.modelCardInfo;
```

Comment:
nit: could collapse this into `const {systemPrompt, temperature, retrievalContexts, modelCardInfo} = useAppSelector(state => state.aichat.previouslySavedAiCustomizations);`?